### PR TITLE
2022.12.0

### PR DESCRIPTION
**Pull request by frenck**
# ⚠️ PR for reviewing, please do not merge.

### CODE REVIEW

File: source/_posts/2022-12-07-release-202212.markdown [hunk 1]
```diff
@@ -0,0 +1,740 @@
...
+
+[Allen Porter]: https://github.com/allenporter
+
+## Bluetooth proxy using Shelly devices
+
+In the September release, we brought [Bluetooth everywhere],
+by adding Bluetooth proxy support, allowing any ESPHome device in your home
+to relay Bluetooth traffic across your home.
+
+This release adds support for using Shelly devices are a remote Bluetooth
```
Comment (puddly) [inline source/_posts/2022-12-07-release-202212.markdown]:
```suggestion
This release adds support for using Shelly devices as a remote Bluetooth
```

File: source/_posts/2022-12-07-release-202212.markdown [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+  Home Assistant to consider those in future features.
+- Using [Twinkly] in your Christmas tree? 🎄 You can now control the effects!
+  Right on time! Thanks, [@Olen]!
+- [MQTT] reached the gold level on the [integration quality scale]! Nice work [@jbouwh]!
+  But that is not all; it now supports MQTT v5 and WebSocket connections too!
+- NO2 and VOC sensor entities are now working with [HomeKit]. Thanks, [@stackia]!
+- [Shelly] is now a [platinum integration]! Thanks, [@thecode], [@bieniu],
+  and [@chemelli74] for your hard work on this integration 🥇
+- [ZHA] now supports the new [Aqara c1 pet feeder]. Thanks [@dmulcahey] for
+  adding it, and thanks [@kirovilya] figuring out the communication!
```
Comment (MartinHjelmare) [inline source/_posts/2022-12-07-release-202212.markdown:233]: for figuring out the communication!

File: source/_posts/2022-12-07-release-202212.markdown [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+[@stackia]: https://github.com/stackia
+[#81217]: https://github.com/home-assistant/core/pull/81217
+
+{% enddetails %}
+
+{% details "Logitech Harmony Hub" %}
+
+The `PowerOff` state of the select entity will be changed to `power_off`.
+
+If you relied on this state value in your automations or script,
```
Comment (MartinHjelmare) [inline source/_posts/2022-12-07-release-202212.markdown:459]: in your automations or scripts,

File: source/_posts/2022-12-07-release-202212.markdown [hunk 4]
```diff
@@ -0,0 +1,741 @@
...
+[#77491]: https://github.com/home-assistant/core/pull/77491
+
+{% enddetails %}
+
+{% details "Mobile Apps" %}
+
+The events fired when the mobile app scanned a Tag had an incorrect device ID;
+This has been corrected.
+
+As a result, the evens now contain a different but correct device ID. If you
```
Comment (MartinHjelmare) [inline source/_posts/2022-12-07-release-202212.markdown:474]: the events now contain

File: source/_posts/2022-12-07-release-202212.markdown [hunk 5]
```diff
@@ -0,0 +1,741 @@
...
+
+[@jbouwh]: https://github.com/jbouwh
+[#82244]: https://github.com/home-assistant/core/pull/82244
+
+{% enddetails %}
+
+{% details "OpenUV" %}
+
+The previously deprecated `openuv.update`, `openuv.update_uv_index`,
+and `openuv.update_protection_data` services have now be removed.
```
Comment (MartinHjelmare) [inline source/_posts/2022-12-07-release-202212.markdown:557]: have now been removed

**Review by MartinHjelmare — COMMENTED** [review]
Nice!

**Pull request merged**